intro-HPC/ch_account.tex

```diff
@@ -259,6 +259,50 @@ \section{Getting ready to request an account}
   \strong{Without your key pair, you won't be able to apply for a personal VSC account.}
 \fi %macORlinux
 
+\iflinux
+
+\subsection{Using an SSH agent (optional)}
+\label{sec:using-ssh-agent-with-openssh}
+  The most recent GNU/Linux distributions include by default an SSH agent 
+  (``gnome-keyring-daemon'' in most cases) to keep and manage the user SSH keys.
+  If you use one of these distributions you \strong{must} include the new key/s into 
+  the SSH manager keyring to be able to connect to the HPC cluster. If not,
+  SSH client will display an error message (see \autoref{ch:preparing-the-environment}) 
+  similar to this:
+  
+  \begin{flattext}
+   Agent admitted failure to sign using the key.
+   Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
+  \end{flattext}
+
+  This could be fixed using the ``ssh-add'' command.
+  You can include the new private key/s identities in your keyring with:
+\begin{prompt}
+%\shellcmd{ssh-add}%
+\end{prompt}
+
+\begin{tip}
+  Without extra options ``ssh-add'' adds any key located at ``~/.ssh''
```

Comment:
did you test this when the user has more then 3 keys? ssh will only try 3 keys and then give up. So it might not be a good idea to add all keys to the agent.

Reply:
We should keep this as simple as possible imho. This document just describes how to generate a ssh key with `ssh-keygen`   without extra options in the previous section, that will create a `id_rsa`  key by default. We can include a lot of examples about `ssh-keygen`  and `ssh-add`   but I think that this is out of the scope of this document, we provide external links if the user wants to try something else. I think that this is enough for a regular user 😄